systests/microprofile/pom.xml

```diff
@@ -33,7 +33,7 @@
     <url>http://cxf.apache.org</url>
     <properties>
         <cxf.geronimo.config.version>1.0</cxf.geronimo.config.version>
-        <cxf.microprofile.rest.client.version>1.2.1</cxf.microprofile.rest.client.version>
+        <cxf.microprofile.rest.client.version>1.3-RC1</cxf.microprofile.rest.client.version>
```

Comment:
Maybe these versions could be consolidated a bit? On master I see:

parent/pom.xml ->  <cxf.microprofile.rest.client.version>1.0.1</cxf.microprofile.rest.client.version>
systests/microprofile/client/jaxrs ->  <cxf.microprofile.rest.client.version>1.2.0</cxf.microprofile.rest.client.version>
systests/microprofile/async/jaxrs -> <cxf.microprofile.rest.client.version>1.2.0</cxf.microprofile.rest.client.version>
rt/rs/microprofile/client -> <mp.rest.client.version>1.2.0</mp.rest.client.version>
systests/microprofile ->  <cxf.microprofile.rest.client.version>1.2.1</cxf.microprofile.rest.client.version>

Reply:
Done.  Thanks for the suggestion!